content/about/index.md

```diff
@@ -64,6 +64,7 @@ The first draft of this specification has been written in collaboration with som
 
 ## Projects Using Conventional Commits
 
+* [Sylva Identity Discovery](https://sylva.pfeister.dev/): Novel OSINT utility that can be used to help link social media identities to their owners.
```

Comment:
We suggest adding the new ones to the bottom of the list, wold you mind @ppfeister ?

Reply:
Done and done!